### DISCUSSION TITLE
installation of @resvg/resvg-js & satori dependencies

### DISCUSSION THREAD

**Discussion by jasongitmail** [etherCorps/sveltekit-og#13] [General]
In 1.2.3, `@ethercorps/sveltekit-og` now requires the installation of `@resvg/resvg-js` and `satori` separately. 

What build errors does that solve? I'm curious to learn to help brainstorm an alternate approach because it would seem to be a more ideal user experience if those were installed as dependencies (ideally) or at least peer dependencies.

**Comment by theetherGit** [comment]
By mistake in last version i forgot to mention to install the peer-dependency. That's all, i made mistakes in last version so currently i'm focusing on my mistakes i made and fixing them. 

And this time i was checking for something and by mistake i replaced peer-dependency by dependency. 

I'm working on a basic docs, so i'll be clearing all the issues before releasing it.

@jasongitmail can you help me with docs?

**Reply by theetherGit** [reply]
@jasongitmail should i change dependency to peer dependency? I think this will resolve this query

**Comment by jasongitmail** [comment]
What do you have in mind? I thought your current docs seemed sufficient, with the excellent Quickstart section. The only thing I'd suggest is to start with Installation and Quickstart sections and move the Changelog below those.

**Reply by theetherGit** [reply]
I feel like readme is a hard to read and nowadays we are used to read programmatically structured docs. That's all

**Reply by theetherGit** [reply]
I'm looking into astro docs template they are good and pretty stable.

**Reply by jasongitmail** [reply]
I can submit a PR to help reorganize it a bit. You can tweak it over time. Personally I wouldn't worry much about a separate docs site, but your call.

**Reply by theetherGit** [reply]
Well it's not seperate docs, i do have other sveltekit related library and i'm thinking of providing docs for that too.

sveltekit-redis-session

**Reply by theetherGit** [reply]
B/w for now if possible can you do the tweaks in the readme? if you can i'll have some time to focus and also be a guide for official docs too.

**Reply by jasongitmail** [reply]
Yep, I submitted a PR to refactor the README. https://github.com/etherCorps/sveltekit-og/pull/14 Should be a good first start and we can keep improving it from here.

**Reply by theetherGit** [reply]
Thank you so much.

**Comment by jasongitmail** [comment]
FWIW, `@ethercorps/sveltekit-og` v1.2.3 still works for me without requiring separate installation of `@resvg/resvg-js` and `satori`. I ran `npm uninstall @ethercorps/sveltekit-og && npm i -D @ethercorps/sveltekit-og` to confirm. Personally I like it this way, it's a good user experience.

**Reply by theetherGit** [reply]
Ok, IDK why it didn't work on mine.

Which adapter are you using?

**Reply by jasongitmail** [reply]
Good question. `import adapter from '@sveltejs/adapter-cloudflare';`

**Reply by theetherGit** [reply]
Ok I'll look into it. It's already too late, i'll check more on it tomorrow.

**Reply by theetherGit** [reply]
`vercel` build doesn't have any issues only `node` adapter causing this.

**Comment by theetherGit** [comment]
> In 1.2.3, `@ethercorps/sveltekit-og` now requires the installation of `@resvg/resvg-js` and `satori` separately.
> 
> What build errors does that solve? I'm curious to learn to help brainstorm an alternate approach because it would seem to be a more ideal user experience if those were installed as dependencies (ideally) or at least peer dependencies.

If you have some more ideas please let me and work on it together.
